src/rules/converters/ban-types.ts

```diff
@@ -1,10 +1,37 @@
 import { RuleConverter } from "../converter";
 
-export const convertBanTypes: RuleConverter = () => {
+export const convertBanTypes: RuleConverter = tslintRule => {
+    type ConvertBanTypeArgument = {
+        message: string;
+    };
+
+    const bannedTypesObj: { [key: string]: ConvertBanTypeArgument | null } = {};
```

Comment:
Small nit, `Record` is now the built-in type that can handle these kinds of types:

```suggestion
    const bannedTypesObj: Record<string, ConvertBanTypeArgument | null> = {};
```